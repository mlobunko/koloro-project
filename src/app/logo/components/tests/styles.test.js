import React from "react";
import { shallow } from "enzyme";
import "enzyme-to-json";

import {
  StyledLogoO,
  StyledLogoWords,
  StyledReactTilt,
  StyledReactTiltMobile
} from "../styles";

describe("StyledLogoO", () => {
  it("renders correctly, landscape true", () => {
    const landscape = true;
    const wrapper = shallow(<StyledLogoO landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
  it("renders correctly, landscape false", () => {
    const landscape = false;
    const wrapper = shallow(<StyledLogoO landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledLogoWords", () => {
  it("renders correctly, landscape true", () => {
    const landscape = true;
    const wrapper = shallow(<StyledLogoWords landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
  it("renders correctly, landscape false", () => {
    const landscape = false;
    const wrapper = shallow(<StyledLogoWords landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledReactTilt", () => {
  it("renders correctly, landscape true", () => {
    const landscape = true;
    const wrapper = shallow(<StyledReactTilt landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
  it("renders correctly, landscape false", () => {
    const landscape = false;
    const wrapper = shallow(<StyledReactTilt landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledReactTiltMobile", () => {
  it("renders correctly, landscape true", () => {
    const landscape = true;
    const wrapper = shallow(<StyledReactTiltMobile landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
  it("renders correctly, landscape false", () => {
    const landscape = false;
    const wrapper = shallow(<StyledReactTiltMobile landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});
