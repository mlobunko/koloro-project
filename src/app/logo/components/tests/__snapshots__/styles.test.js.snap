// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledLogoO renders correctly, landscape false 1`] = `
<div
  className="sc-bdVaJa xHRFh"
/>
`;

exports[`StyledLogoO renders correctly, landscape true 1`] = `
<div
  className="sc-bdVaJa ktNRbW"
/>
`;

exports[`StyledLogoWords renders correctly, landscape false 1`] = `
<div
  className="sc-bwzfXH dKwhYy"
/>
`;

exports[`StyledLogoWords renders correctly, landscape true 1`] = `
<div
  className="sc-bwzfXH gQmgmc"
/>
`;

exports[`StyledReactTilt renders correctly, landscape false 1`] = `
<t
  className="sc-htpNat jAURfV"
  landscape={false}
  options={
    Object {
      "max": 40,
      "mobile": false,
      "perspective": 5000,
      "position-base": "window",
      "reset": true,
      "reverse": true,
    }
  }
/>
`;

exports[`StyledReactTilt renders correctly, landscape true 1`] = `
<t
  className="sc-htpNat egoDtD"
  landscape={true}
  options={
    Object {
      "max": 40,
      "mobile": false,
      "perspective": 5000,
      "position-base": "window",
      "reset": true,
      "reverse": true,
    }
  }
/>
`;
